communication/slack-config/sig-node/config.yaml

```
@@ -1,6 +1,6 @@
 channels:
   - name: sig-node
-  - name: sig-node-bug-scrub
+  - name: sig-node-ci
```

```suggestion
  - name: sig-node-bug-scrub
    archived: true
  - name: sig-node-ci
```
The channel needs to be flagged as archived and a new one added